decidim-admin/app/commands/decidim/admin/create_attachment.rb

```diff
@@ -28,7 +28,7 @@ def call
           @attachment.save!
           broadcast(:ok)
         else
-          if @attachment.errors.has_key? :file
+          if @attachment.errors.key? :file
```

Comment:
I think there are smaller number of instances because we ran `rubocop -a` and autocorrected it in the past, but no because it was preferred.